operator/hack/boilerplate.go.txt

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
is it possible to drop this file and rely on boilerplate files under hack/ ?

Reply:
🙂  Yup. Taken care of in the next PR #4126